src/aiida/cmdline/commands/cmd_node.py

```diff
@@ -372,7 +372,7 @@ def _perform_delete():
         from aiida.orm.utils.remote import clean_mapping_remote_paths, get_calcjob_remote_paths
         from aiida.tools.graph.graph_traversers import get_nodes_delete
 
-        backend = get_manager().get_backend()
+        backend = get_manager().get_profile_storage()
```

Comment:
>   /home/hollas/atmospec/aiida-core/src/aiida/manage/manager.py:251: AiidaDeprecationWarning: get_backend() is deprecated, use get_profile_storage() instead (this will be removed in v3)
    warn_deprecation('get_backend() is deprecated, use get_profile_storage() instead', version=3)

Reply:
After adjusting the stacklevel

>   /home/hollas/atmospec/aiida-core/src/aiida/cmdline/commands/cmd_node.py:375: AiidaDeprecationWarning: get_backend() is deprecated, use get_profile_storage() instead (this will be removed in v3)
    backend = get_manager().get_backend()